templates/react-native-template/template/index.js

```diff
@@ -17,4 +17,6 @@ const App = () =>
     <AppWrapperNonSync />
   );
 
+export default App;
+
 AppRegistry.registerComponent(appName, () => App);
```

Comment:
Is there something importing `App`? Seemed to only be passed to `AppRegistry` before.

Reply:
The Jest test